code/game/objects/items/storage/belt.dm

```diff
@@ -16,6 +16,20 @@
 	///TRUE Means that it closes a flap over its contents, and therefore update_icon should lift that flap when opened. If it doesn't have _half and _full iconstates, this doesn't matter either way.
 	var/flap = TRUE
 
+/obj/item/storage/belt/gun/flaregun/dump_into(obj/item/storage/Main, mob/user)
```

Comment:
Two things:
- `Main` makes no sense to be the variable name; this is kinda more confusing than just a random letter like it used to be.
- We use snake_case; not camelCase or PascalCase. Your variable here is PascalCase. https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#type-paths-must-be-snake-case

So something more like `obj/item/storage/origin_storage` would be more appropriate.

Reply:
My bad put it there as a placeholder i replace it with origin_storage like you suggest.